app/org/sagebionetworks/bridge/models/schedules/Activity.java

```diff
@@ -145,9 +141,10 @@ public Activity build() {
             if (guid == null) {
                 guid = BridgeUtils.generateGuid();
             }
-            Activity activity = new Activity(label, labelDetail, guid, task, survey, response);
-            Validate.entityThrowingException(VALIDATOR, activity);
-            return activity;
+            // Activity is validated as part of a schedule, and requires task identifiers 
+            // for the study being validated. We can't validate again, here, during construction 
+            // without adding taskIdentifiers to the builder, which is strange.
+            return new Activity(label, labelDetail, guid, task, survey, response);
```

Comment:
I think there's still a lot of value in making Activity self-validating, as far as internal validation goes. This way, we can ensure basic things, like label is non-null, etc.

For validating we have valid task IDs, it would be better to have that as a separate step.